File: src/renders/habitats/habitat.cpp
/* NIM/Nama  : 13515AAA / AAA       */
/* File      : Habitat.cpp              */

#include "habitat.h"

using namespace std;

Habitat::Habitat() : Cell()
{
}

Habitat::Habitat(char _id, Color _color) : Cell(_id,_color)
{
}

Habitat::Habitat(const Habitat& H) : Cell (H)
{
}

Habitat::~Habitat()
{
}

Habitat& Habitat::operator=(const Habitat& H)
{
	Cell::operator=(H);
	return *this;
}

File: Habitat.java
/* NIM/Nama : 13515078 / Veren Iliana Kurniadi */
/* File : Habitat.java */

import java.util.*;
import java.lang.*;

abstract class Habitat extends Cell {
  /** 
   * Constructor.
   */
  public Habitat() {
    super();
  }
  /** 
   * Constructor dengan parameter.
   * Menciptakan Habitat dengan parameter id dan color
   * @param id char renderable
   * @param color warna renderable
   */
  public Habitat(char _id, Color _color) {
    super(_id,_color);
  }
}